samples/msal-browser-samples/HybridSample/views/layout.hbs

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>{{title}}</title>
+    <link rel='stylesheet' href='/stylesheets/style.css' />
+    <script src='/msal/msal-browser.js'></script>
+    <script type="text/javascript">
+        const msalInstance = new msal.PublicClientApplication({
+            auth: {
+                clientId: "{{clientId}}",
+                redirectUri: "http://localhost:3000/client-redirect",
+                authority: "https://login.microsoftonline.com/common"
+            }
+        })
+
+        msalInstance.handleRedirectPromise()
```

Comment:
`handleRedirectPromise` shouldn't be needed in the hybrid scenario, right?

Reply:
While the hybrid auth code should greatly reduce or eliminate the need for interaction in the first boot flows, we'll still want to document that apps need to have error handling and include interactive flows as fallbacks.